content/docs/2.15/concepts/scaling-deployments.md

```diff
@@ -21,270 +23,30 @@ For example, if you wanted to use KEDA with an Apache Kafka topic as event sourc
-
----
-#### idleReplicaCount
-
-```yaml
-  idleReplicaCount: 0   # Optional. Default: ignored, must be less than minReplicaCount
-```
-
-> 💡 **NOTE:** Due to limitations in HPA controller the only supported value for this property is 0, it will not work correctly otherwise. See this [issue](https://github.com/kedacore/keda/issues/2314) for more details.
->
-> In some cases, you always need at least `n` pod running. Thus, you can omit this property and set `minReplicaCount` to `n`.
->
-> **Example** You set `minReplicaCount` to 1 and `maxReplicaCount` to 10. If there’s no activity on triggers, the target resource is scaled down to `minReplicaCount` (1). Once there are activities, the target resource will scale base on the HPA rule. If there’s no activity on triggers, the resource is again scaled down to `minReplicaCount` (1).
-
-If this property is set, KEDA will scale the resource down to this number of replicas. If there's some activity on target triggers KEDA will scale the target resource immediately to `minReplicaCount` and then will be scaling handled by HPA. When there is no activity, the target resource is again scaled down to `idleReplicaCount`. This setting must be less than `minReplicaCount`.
-
-**Example:** If there's no activity on triggers the target resource is scaled down to `idleReplicaCount` (0), once there is an activity the target resource is immediately scaled to `minReplicaCount` (10) and then up to `maxReplicaCount` (100) as needed. If there's no activity on triggers the resource is again scaled down to `idleReplicaCount` (0).
-
----
-#### minReplicaCount
-```yaml
-  minReplicaCount: 1   # Optional. Default: 0
-```
-
-Minimum number of replicas KEDA will scale the resource down to. By default, it's scale to zero, but you can use it with some other value as well.
-
----
-#### maxReplicaCount
-```yaml
-  maxReplicaCount: 100 # Optional. Default: 100
-```
-
-This setting is passed to the HPA definition that KEDA will create for a given resource and holds the maximum number of replicas of the target resource.
-
----
-#### fallback
-```yaml
-  fallback:                                          # Optional. Section to specify fallback options
-    failureThreshold: 3                              # Mandatory if fallback section is included
-    replicas: 6                                      # Mandatory if fallback section is included
-```
-
-The `fallback` section is optional. It defines a number of replicas to fall back to if a scaler is in an error state.
+# Features
```

Comment:
Given this used to be hear, maybe we need `# Specification` which simply points to the reference as mentioned above?

Reply:
@tomkerkhove We reference the spec at the top of this page. Maybe the thing to do is to make it more prominent there.

Also, do the same on the Scaling Jobs page.